src/styles/components/card.scss

```diff
@@ -19,7 +19,7 @@ $block: ns(card);
     $tn-card-media-width: 75px !default;
     $tn-card-color-border: tn-color(neutral,2) !default;
     $tn-card-color-disabled: tn-color(text,3) !default;
-    $tn-card-background-color: tn-color(inverse) !default;
+    $tn-card-background-color: tn-color(text-inverse) !default;
```

Comment:
The color group names **are identified by use**. That's why using colors from the text-inverse group for background needs to be changed.

The color system as it is is scalable, no need to narrow it for now.

I'll talk with Leo and see how we gonna add the new color